src/show/pages/public/Home/index.jsx

```diff
@@ -1,12 +1,22 @@
 import axios from 'axios';
 import { useEffect, useState } from 'react';
 
-import { ConstructorCard, DriverCard } from '../../../components';
+import {
+  ConstructorCard,
+  DriverCard,
+  UpcomingEvent
+} from '../../../components';
 
 const Home = () => {
   const [constructors, setConstructors] = useState([]);
+  const [event, setEvent] = useState([]);
```

Comment:
One single event, so the initial state cannot be an empty array.

Reply:
okay i agree

Reply:
Still has to have an initial state. What would be suitable for this?